app/forms/custom_fields.py

```diff
@@ -25,7 +25,7 @@ def process_formdata(self, valuelist):
 
         if valuelist:
             try:
-                self.data = int(valuelist[0])
+                self.data = int(valuelist[0].replace(',', ''))
```

Comment:
We should use Babel to provide us with the thousand separator for the locale (get_group_symbol()). This is also used for CustomDecimal and in validators.py